pinot-core/src/test/java/org/apache/pinot/core/common/datatable/DataTableSerDeTest.java

```diff
@@ -123,36 +123,142 @@ public void testException(int dataTableVersion)
   }
 
   @Test(dataProvider = "versionProvider")
-  public void testEmptyStrings(int dataTableVersion)
+  public void testEmptyValues(int dataTableVersion)
       throws IOException {
     DataTableBuilderFactory.setDataTableVersion(dataTableVersion);
     String emptyString = StringUtils.EMPTY;
     String[] emptyStringArray = {StringUtils.EMPTY};
+    ByteArray emptyBytes = new ByteArray(new byte[0]);
+    for (int numRows = 0; numRows < NUM_ROWS; numRows++) {
+      testEmptyValues(new DataSchema(new String[]{"STR_SV", "STR_MV"}, new DataSchema.ColumnDataType[]{
```

Comment:
i dont know if this is possible but lets also test empty `OBJECT` type?

Reply:
We don't really use `OBJECT` type anywhere and what is an empty `OBJECT` if not null?